readthedocs/projects/models.py

```diff
@@ -144,10 +144,11 @@ class AddonsConfig(TimeStampedModel):
     Everything is enabled by default.
     """
 
-    DOC_DIFF_DEFAULT_ROOT_SELECTOR = "[role=main]"
-    LINKPREVIEWS_DEFAULT_ROOT_SELECTOR = "[role=main] a.internal"
-    LINKPREVIEWS_DOCTOOL_NAME_CHOICES = (
+    DEFAULT_ROOT_SELECTOR = ":is([role=main],main,div.body,div.document)"
```

Comment:
Is `is()` required? `querySelectorAll("[role=main], main, div.body, div.document")` should be the same. Does `css_first` support this?